mobile/android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">mobile</string>
+     <string name="web_client_id">807234752761-92gmhcgop4pcqflhdiftfhtt4bdhr3a4.apps.googleusercontent.com</string>
```

Comment:
is that normal that the client id is not an env variable ?

Reply:
+1